bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableInstanceName.java

```diff
@@ -96,4 +102,29 @@ public String toTableNameStr(String tableId) {
   public BigtableTableName toTableName(String tableId) {
     return new BigtableTableName(toTableNameStr(tableId));
   }
+
+  /**
+   * Creates a fully qualified cluster within the current instance.
```

Comment:
Creates a fully qualified cluster *name* within the current instance.

Does this work if instanceId is "-"?